### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by flands**
- Drop `Instrumentation` from name
- Add more build badges
- Add link to documentation based on release
- Add information about B3 and Jaeger Thrift to README
- Update FAQ
- Update Versioning

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,13 +11,19 @@
 </p>
 
 <p align="center">
+  <img alt="Beta" src="https://img.shields.io/badge/status-beta-informational?style=for-the-badge">
+  <a href="https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/tag/v1.2.0">
+    <img alt="OpenTelemetry Java Instrumentation Version" src="https://img.shields.io/badge/otel-1.2.0-blueviolet?style=for-the-badge">
+  </a>
+  <a href="https://github.com/signalfx/splunk-otel-java-instrumentation/releases">
```
Comment (iNikem) [inline README.md:18]:
```suggestion
  <a href="https://github.com/signalfx/splunk-otel-java/releases">
```

File: VERSIONING.md [hunk 2]
```diff
@@ -1,31 +1,58 @@
-# Splunk Distribution of OpenTelemetry Java Instrumentation Versioning
+# Splunk Distribution of OpenTelemetry Java Versioning
+
+Splunk Distribution of OpenTelemetry Java pulls in the the OpenTelemetry Java
```
Comment (iNikem) [inline VERSIONING.md:3]: Are you sure that "OpenTelemetry Java" is better in this context than "OpenTelemetry Java Instrumentation"

File: VERSIONING.md [hunk 3]
```diff
@@ -1,31 +1,58 @@
...
+Instrumentation repository. As a result it cannot and will not mark anything
+stable, meaning supported without breaking changes, unless OpenTelemetry has
+marked it as stable or it is not specific to OpenTelemetry and is marked stable
+from by some other means (for example the GDI specification).
```
Comment (iNikem) [inline VERSIONING.md:7]:
```suggestion
by some other means (for example the GDI specification).
```

File: VERSIONING.md [hunk 4]
```diff
@@ -1,31 +1,58 @@
...
+before introducing the change, so that customers can prepare accordingly.
 
 All the changes are to be communicated in the [changelog](CHANGELOG.md).
 
 ### Old versions support
 
-Splunk is committed to support the customers using this distribution. All major versions will get critical (eg security) patches for **one year** after the release date. 
+Splunk is committed to support the customers using this distribution. All major
+versions will get critical (for example security) patches for **one year**
+after the release date. Feature development will stop on a major release once a
```
Comment (iNikem) [inline VERSIONING.md:51]: After the initial release of that major version or one year after the release of the next major version?

File: docs/faq.md [hunk 5]
```diff
@@ -1,14 +1,18 @@
...
-
-## How often do you release?
-
-We strive to release the Splunk distribution at most 2 working days after the
-[upstream project](https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases)
-releases. We release a new version every 2-4 weeks.
+- **Can upstream opentelemetry-java or opentelemetry-java-instrumentation be
+  used instead?** Definitely, however Splunk only provides best-effort support.
+- **What’s different between Splunk Distribution of OpenTelemetry Java and
+  OpenTelemetry Java?** Supported by Splunk, better defaults for Splunk
```
Comment (iNikem) [inline docs/faq.md:6]: OpenTelemetry Java does not provide auto-instrumention. Isn't it important here?

File: VERSIONING.md [hunk 6]
```diff
@@ -1,31 +1,58 @@
...
+
+- OpenTelemetry Java Instrumentation breaking change - please see [upstream
+  repository](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/VERSIONING.md)
+  for their versioning policy
+- Change in a Splunk-managed configuration property or code
+
+In the latter case, we will first **deprecate** (at least one release earlier)
+before introducing the change, so that customers can prepare accordingly.
 
 All the changes are to be communicated in the [changelog](CHANGELOG.md).
```
Comment (mateuszrzeszutek) [inline VERSIONING.md:45]: Should we explicitly mention that unstable features (like metrics) MAY be changed in a backwards incompatible way between `MINOR` releases?

File: README.md [hunk 7]
```diff
@@ -11,13 +11,19 @@
...
+    <img alt="GitHub release (latest SemVer)" src="https://img.shields.io/github/v/release/signalfx/splunk-otel-java?include_prereleases&style=for-the-badge">
+  </a>
+  <a href="https://mvnrepository.com/artifact/io.opentelemetry/opentelemetry-api">
+    <img alt="Maven Central" src="https://img.shields.io/maven-central/v/io.opentelemetry/opentelemetry-api?style=for-the-badge">
+  </a>
```
Comment (mateuszrzeszutek) [inline README.md:23]: Do we really need that? I find it confusing that we link to opentelemetry-api in the badge (which is one of the first things that you see when you open the page) when manual instrumentation is a more advanced and not that common use case.

File: README.md [hunk 8]
```diff
@@ -71,19 +87,20 @@ curl -L https://github.com/signalfx/splunk-otel-java/releases/latest/download/sp
...
 
-Enable the Java agent by adding the `-javaagent` flag to the runtime JVM parameters.
-For example, if the runtime parameters were:
+Enable the Java agent by adding the `-javaagent` flag to the runtime JVM
+parameters and set the `service.name`. For example, if the runtime parameters
+were:
 
 ```bash
-java -jar myapp.jar https://google.com
+java -jar myapp.jar https://splunk.com
```
Comment (mateuszrzeszutek) [inline README.md:95]: 👍

File: VERSIONING.md (hunk 2 above)
Comment (flands) [reply VERSIONING.md:3]: Right now, Splunk standardized on Splunk Distribution of OpenTelemetry <language>. Given both auto and manual is possible likely OK.

**Review by mateuszrzeszutek — APPROVED** [review]

**Pull request merged**